tests/phpunit/tests/functions/wpuploadbits.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Test the wp_upload_bits function
+ *
+ * @group Functions
+ * @group Upload
+ * @covers ::wp_upload_bits
+ */
+
+class tests_wp_upload_bits extends WP_UnitTestCase {
+
+	/**
+	 * @expectedDeprecated
+	 * @return void
+	 *
+	 */
+	public function test_second_parm_present() {
+		$this->expectExceptionMessage( 'Function wp_upload_bits was called with an argument that is deprecated since version 2.0.0 with no alternative available.' );
+		$this->expectDeprecation();
+		$this->expectException( 'PHPUnit\Framework\Error\Deprecated' );
+		wp_upload_bits( 'filename.txt', 'not_null', 'bits' );
+	}
+	/**
+	 * return error if no filename pass in
+	 * @expectedDeprecated
+	 **/
+	public function test_no_name_present() {
+		$this->expectDeprecation();
```

Comment:
```suggestion
	 **/
	public function test_no_name_present() {
```
This test method will not cause a deprecation notice.